tower-batch/Cargo.toml

```diff
@@ -15,7 +15,7 @@ tracing-futures = "0.2.5"
 futures = "0.3.12"
 
 [dev-dependencies]
-ed25519-zebra = "2.1.0"
+ed25519-zebra = "2"
```

Comment:
This is a dev dependency for the `tower-batch` unit tests.

It's not part of the consensus rule change, so it should stay at `2.1.0`.